aten/src/ATen/TensorNames.cpp

```diff
@@ -83,7 +84,7 @@ TensorNames& TensorNames::unifyFromRightInplace(const TensorNames& other, const
   return *this;
 }
 
-void TensorNames::append(TensorName&& name) {
+void TensorNames::append(TensorName name) {
   names_.emplace_back(name);
```

Comment:
Should name be moved into the emplace_back or is trivially copyable?

Reply:
It is trivially copyable.